extensions/github-search/src/run-saved-search.tsx

```diff
@@ -0,0 +1,41 @@
+import { List, ActionPanel, Action } from '@raycast/api';
+import { useLocalStorage } from '@raycast/utils';
+import type { ReusableFilter, SavedSearch } from './types';
+import { useFrecencySorting } from '@raycast/utils';
+import { buildUrl } from './search';
+import { launchCommand, LaunchType } from '@raycast/api';
+
+export default function SavedSearches() {
+  const { value: savedSearches = [] } = useLocalStorage<SavedSearch[]>('saved-searches', []);
+  const { value: reusableFilters = [] } = useLocalStorage<ReusableFilter[]>('repo-filters', []);
+  const { data: sortedSearches, visitItem } = useFrecencySorting(savedSearches);
+
+  return (
+    <List>
+      {sortedSearches.length === 0 ? (
+        <List.EmptyView
+          title="No Saved Searches"
+          description="Create your first saved search in the main GitHub Search view"
+          actions={
+            <ActionPanel>
+              <Action title="Open GitHub Search" onAction={() => launchCommand({ name: 'search', type: LaunchType.UserInitiated })} />
```

Comment:
Should probably be `async` with `await` - this can potential crash the extension so we also need it in a try catch

Reply:
Okay :)